packages/subgraph/docker/hardhat.Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:18
```

Comment:
Optional: Create a new user and switch to it to prevent the command from running as root

Check other Dockerfiles created by the DevOps team as reference

Reply:
I copied this from another Dockerfile ```RUN addgroup -S app && adduser -S app -G app -s /sbin/nologin && mkdir -p /app && chown -R app:app /app```, but I'm having a hard time to get it working for me. Whatever I do, this line fails saying `ambiguous options`.